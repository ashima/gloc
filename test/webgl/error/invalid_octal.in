#if 9 == 09 // invalid octal is a cardinal sin

#endif
void main() {}
