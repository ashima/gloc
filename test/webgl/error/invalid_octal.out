File 0, line 1, col 9 - 11:
invalid octal constant "09"

Fatal: unrecoverable preprocessor parse error (3)
